e2e/native-esm/wasm-bindgen/index.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+// folder source: https://github.com/rustwasm/wasm-bindgen/tree/4f865308afbe8d2463968457711ad356bae63b71/examples/hello_world
+// docs: https://rustwasm.github.io/docs/wasm-bindgen/examples/hello-world.html
+
+import * as wasm from './index_bg.wasm';
```

Comment:
Unused import?

Reply:
It looks so, yes. All three files in this folder are auto-generated and I’ve decided to keep them as is. This will be helpful in the future if we need to re-generate the example again.

Maybe this like has its meaning actually. It makes sure that the wasm file is imported before JS, which may affect the result. `index_bg.wasm` and `index_bg.js` import stuff from each other.